.swiftlint.yml

```diff
@@ -1,56 +1,62 @@
 # .swiftlint.yml
 #
-# Useful rules reference: https://realm.github.io/SwiftLint/rule-directory.html
+# Goals:
+#   - Decrease number of `disabled_rules` that we agree with as a team
+#   - Increase number of `opt_in_rules` that we agree with as a team
+#   - Increase number of `custom_rules` that we agree with as a team
+#   - Avoid rule customization unless increasing strictness that we agree with as a team
+#
+# Useful rules reference: https://github.com/realm/SwiftLint/blob/master/Rules.md
 
 disabled_rules:
+  - cyclomatic_complexity       # This finds some hairy methods
+  - file_length
+  - force_cast
+  - function_body_length        # Interesting overlap between these violations and cyclomatic_complexity
   - line_length
+  - operator_whitespace
+  - todo
   - type_body_length
+  - void_function_in_ternary
+  - force_try
+  - opening_brace
+  - closing_brace
+  - for_where
+  - function_parameter_count
+  - multiple_closures_with_trailing_closure
 
 opt_in_rules:
-  - attributes
-  - closing_brace
   - colon
   - comma
   - control_statement
+  - discouraged_object_literal
   - empty_count
-  - file_header
-  - force_try
+  # - force_unwrapping  # Enable one day...
   - joined_default_parameter
   - leading_whitespace
   - legacy_cggeometry_functions
   - legacy_constant
   - multiline_parameters
-  - nesting
-  - opening_brace
```

Comment:
disabled because it won't allow certain patterns that are recommended by swift-format to break lines, like
```
if
  longCondition1
  && longCondition2
{
  body
}
```
This rule would force
```
if
  longCondition1
  && longCondition2 {
  body
}
```
Which leaves the body at the same level as the condition, making it unreadable